src/android/com/silkimen/cordovahttp/CordovaHttpUpload.java

```diff
@@ -60,6 +65,23 @@ protected void sendBody(HttpRequest request) throws Exception {
 
         request.part(uploadName, fileName, mimeType, inputStream);
       }
+
+      if (hasProgressHandler) {
+        request.progress((transferred, total) -> {
```

Comment:
Please don't use Java 8 features (lambda expression in this case), because this will break support for old Cordova projects still configured to run with Java 7.

Reply:
I've removed the lambda, please check if it's ok now